examples/get_iss_breast_data.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import io
+import os
+from typing import IO, Tuple
+
+from skimage.io import imread, imsave
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Indices
+from starfish.util.argparse import FsExistsType
+
+SHAPE = 1044, 1390
+
+
+class IssCroppedBreastTile(FetchedTile):
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return SHAPE
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    @staticmethod
+    def crop(img):
+        crp = img[40:1084, 20:1410]
+        return crp
+
+    def tile_data_handle(self) -> IO:
```

Comment:
i think this is supposed to be a `@property`. it worked before due to a fluke, but the new slicedimage api will require it.

you can make the change now or i can make it when i land the slicedimage change.

Reply:
will fix